_component_design/selector-strip.md

```diff
@@ -18,15 +18,81 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Overview
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Selector strip types
+
+There are two types of selector strips:
+
+- **Single select strip**: A single-select strip has rounded corners and functions like a radio; the user can only select one option. Selecting another option deselects the previously selected option.
```

Comment:
s/`like a radio; the user`/`like a radio as the user` (normally, I'd use a colon, but the style guide says to avoid them in sentences.)